docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+---
+version: '2'
+
+services:
+  virtualpdu_1:
+    #image: "internap/virtualpdu:latest"
```

Comment:
Because you can't have both "build" and "image".  It was simply to let the reader know that he could rely on the image form dockerhub if he wants to.